src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -36,7 +48,61 @@ use systems::{
     },
 };
 
+struct A320CargoDoorFactory {}
+impl A320CargoDoorFactory {
+    fn a320_cargo_door_actuator(rigid_body: &LinearActuatedRigidBodyOnHingeAxis) -> LinearActuator {
+        LinearActuator::new(
+            rigid_body.max_linear_distance_to_anchor(),
+            rigid_body.min_linear_distance_to_anchor(),
```

Comment:
@crocket63 reopening this one. The intent is to replace the argument of `LinearActuator`, otherwise this whole exercise wasn't really useful 😀.

```rust
    fn a320_cargo_door_actuator(bounded_linear_length: &impl BoundedLinearLength) -> LinearActuator {
        LinearActuator::new(
            rigid_body
            rigid_body
            2,
            Length::new::<meter>(0.04422),
            Length::new::<meter>(0.03366),
            VolumeRate::new::<gallon_per_second>(0.01),
            600000.,
            15000.,
            500.,
        )
    }

impl LinearActuator {
    const DEFAULT_I_GAIN: f64 = 0.2;
    const DEFAULT_P_GAIN: f64 = 0.05;

    #[allow(clippy::too_many_arguments)]
    pub fn new(
        bounded_linear_length: &impl BoundedLinearLength,
        number_of_actuators: u8,
        bore_side_diameter: Length,
        rod_diameter: Length,
        max_flow: VolumeRate,
        fluid_compression_spring_constant: f64,
        fluid_compression_damping_constant: f64,
        active_hydraulic_damping_constant: f64,
    ) -> Self {
        let total_travel = bounded_linear_length.max() - bounded_linear_length.min();
        // ...
    }
}
```